models/core/sushiswap_lp_actions.sql

```diff
@@ -0,0 +1,127 @@
+{{
+    config(
+        materialized='incremental',
+        unique_key='log_id',
+        tags=['core', 'sushiswap_lp_actions'],
+        cluster_by=['log_id']
+        )
+}}
+
+
+with 
+
+events as (

```

Comment:
seperate this into 2 CTE's, one to just get logs, then do your subselect in the 2nd.

put the selection CTE's on top.

Reply:
So I'll make a new CTE with the incremental load filter then use the logs in the events CTE, is that what you mean?

Reply:
yup